src/dataaccess/activity.js

```diff
@@ -0,0 +1,19 @@
+import connection from "../db/connection";
+
+const getActivities = async (req, res) => {
+  const [result] = await connection
+    .promise()
+    .query("SELECT * FROM `Actividades`");
+
+  return result;
+};
+
+const getActivitiesId = async (req, res) => {
+  const [result] = await connection
+    .promise()
+    .query("SELECT * FROM `Actividades` WHERE `id` = ?", [req.params.id]);
+
+  return result;
+};
```

Comment:
Esto solo debería recibir un Id

```suggestion
const getActivitiesId = async ({id}) => {
  const [result] = await connection
    .promise()
    .query("SELECT * FROM `Actividades` WHERE `id` = ?", [id]);

  return result;
};
```